packages/okta-react-native/__tests__/token-client-util.test.js

```diff
@@ -11,10 +11,7 @@
  */
 
 import {
-  requests,
```

Comment:
Probably something for later, but is there linting for this package?

Reply:
Unfortunately not within the `okta-react-native` package.

I'm planning to bring `estlint` out to the root of the repo, so we can run it against all packages in a future PR.